css/home.css

```diff
@@ -0,0 +1,87 @@
+.grid-layout {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-areas: "header header"
+                          "main-left main-right"
+                          "section-bottom section-bottom";
```

Comment:
try writing the grid areas below like,
grid-template-areas: 
    "header header"
    "main-left main-right"
    "section-bottom section-bottom";
so it will look better

Reply:
done